### ISSUE TITLE
PHP 8

### ISSUE THREAD

**Issue by slightlychilled** [croogo/croogo#1003]
Do we have a version of croogo that supports php 8+?

**Comment by mihau1** [comment]
I dont think so. There is 5.0.0 beta... but nobody work on it i guess https://croogo.org/blog/croogo-500-beta1-released

**Comment by slightlychilled** [comment]
So, it would be fair to say, this project is officially retired.
